src/app/components/reorderList.js

```diff
@@ -0,0 +1,72 @@
+import todoList from '../utils/todoList.js';
+import deployList from './deployList.js';
+import completion from './completion.js';
+
+class ReorderList {
+  targetsArray = [];
+
+  droppedOn;
+
+  startIndex;
+
+  taskDragged;
+
+  selectTargets = () => {
+    const startTarget = document.querySelector('.todo-list__start-target');
+    const targets = Array.from(document.querySelectorAll('.todo-list__target'));
+    this.targetsArray = [startTarget].concat(targets);
+  };
+
+  dragStart = (li) => {
```

Comment:
- [ ] Kindly check the issues with the start drag function, because when I hold on to drag the item, is disappears from both the tasks array and the local storage. Please double check on that issue 😊🙏🏻

Reply:
I'm not sure I fix the issue. I couldn't reproduce the bug in firefox or chromium based explorers. This functionality has no support for mobiles and I don't think support for mobiles is in requirements.

I did a couple of changes still in the logic to ensure that if a task is removed from the array, it's added it again by placing both actions in the same event.